cmd/analyzer/main_test.go

```diff
@@ -25,6 +25,18 @@ func TestMain(t *testing.T) {
 		name string
 		args string
 	}{
+		// aws
+		{
+			name: "aws_all_endpoints",
+			args: "report endpoints --output-file aws.txt -c ../../pkg/awsvpc/examples/input/input_config_object.json --format txt",
+		},
+		/*
+			{
+				name: "aws_explain",
```

Comment:
why is this test added in comment? should be removed?

Reply:
it will be available once nacl is added

Reply:
so for now I'd remove it from this branch